.github/SECURITY.md

```diff
@@ -16,7 +16,7 @@ We will keep the details of your security vulnerability report private and only
 
 To report a vulnerability, please *DO NOT* open a public Issue or Pull Request.
 
-Please email your security vulnerability report to security@openmage.org along with your Github user name so that once we create a security advisory you may be added to it as a collaborator for further review.
+Please email your security vulnerability report to security@openmage.org along with your GitHub user name so that once we create a security advisory you may be added to it as a collaborator for further review.
```

Comment:
```suggestion
Please email your security vulnerability report to security@openmage.org along with your GitHub username so that once we create a security advisory you may be added to it as a collaborator for further review.
```